### PR TITLE
Moved APM stats graph into separate guide

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?

Move out the APM stats graph section out to a separate guide

### Motivation

[DOCS-5289](https://datadoghq.atlassian.net/browse/DOCS-5289)

### Additional Notes

Ready to merge after review.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5289]: https://datadoghq.atlassian.net/browse/DOCS-5289?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/dashboards/guide/apm-stats-graph.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Configuring an APM stats graph
```
Comment (alai97) [inline content/en/dashboards/guide/apm-stats-graph.md:2]:
```suggestion
title: Configuring An APM Stats Graph
```

File: content/en/dashboards/guide/apm-stats-graph.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+kind: guide
+disable_toc: false
+aliases:
+- /dashboards/querying/#configuring-an-apm-stats-graph
+further_reading:
+- link: "/dashboards/querying/"
+  tag: "Documentation"
+  text: "Querying graphs"
```
Comment (alai97) [inline content/en/dashboards/guide/apm-stats-graph.md:10]:
```suggestion
  text: "Learn how to query graphs"
```

File: content/en/dashboards/guide/apm-stats-graph.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+To configure your graph using APM stats data, follow these steps:
```
Comment (alai97) [inline content/en/dashboards/guide/apm-stats-graph.md:13]:
```suggestion
## Overview

To configure your graph using APM stats data, follow these steps:
```

File: content/en/dashboards/guide/apm-stats-graph.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+1. [Select your visualization](#select-your-visualization) (same as for Metrics)
+2. [Choose your level of detail](#level-of-detail)
+3. [Choose your parameters](#apm-stats-parameters)
+4. [Title the graph](#create-a-title) (same as for Metrics)
```
Comment (alai97) [inline content/en/dashboards/guide/apm-stats-graph.md:18]: I believe these steps are repeated in https://github.com/DataDog/documentation/pull/18005/files#diff-62c7e5fe878506604440adcc2c67d8ecec46dade159d834db0888bc3d4d14791R35-R41? Should they be permanent links with anchors instead, or do you want to move the content out of the Querying doc page into this guide?

File: content/en/dashboards/guide/apm-stats-graph.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+### Level of detail
+Choose what level of detail you want to see statistics for: one or more services, resources, or spans. (Not all of these are available for every widget type.)
```
Comment (alai97) [inline content/en/dashboards/guide/apm-stats-graph.md:21]:
```suggestion
Choose what level of detail you want to see statistics for: one or more services, resources, or spans. Not all of these are available for every widget type.
```

File: content/en/dashboards/guide/apm-stats-graph.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+### APM stats parameters
+Select the following parameters from the graphing editor: Environment (`env`), Primary tag (`primary_tag`), Service (`service`), and Operation name (`name`).
+
+If your level of detail is resource or span, some widget types also require you to select a Resource name (`resource`) to narrow the scope of your query.
```
Comment (alai97) [inline content/en/dashboards/guide/apm-stats-graph.md:26]:
```suggestion
If your level of detail is `resource` or `span`, some widget types also require you to select a Resource name (`resource`) to narrow the scope of your query.
```

File: content/en/dashboards/guide/apm-stats-graph.md (hunk 4 above)
Comment (estherk15) [reply content/en/dashboards/guide/apm-stats-graph.md:18]: Ooh, thanks for catching this, copy pasta error. I'll link out to the Querying page.

**Pull request merged**